libraries/botbuilder-core/tests/storageBaseTests.js

```diff
@@ -94,7 +94,7 @@ class StorageBaseTests {
     }
 
     static async handleCrazyKeys(storage) {
-        const key = `!@#$%^&*()~/\\><,.?';\"\`~`;
+        const key = '!@#$%^&*()~/\\><,.?\';"`~';
```

Comment:
I think we already had the discussion about this change, i believe it was ok, right?

Reply:
Yes, it's not a regex, just a string.